images/bootstrap/Makefile

```diff
@@ -20,8 +20,8 @@ all: build
 build:
 	bazel build //images/bootstrap/barnacle:barnacle --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64
 	cp ./../../bazel-bin/images/bootstrap/barnacle/linux_amd64_pure_stripped/barnacle ./barnacle/barnacle
-	docker build --build-arg IMAGE_ARG=$(IMG):$(TAG) -t $(IMG):$(TAG) .
-	rm ./barnacle/barnacle
+	docker build --pull --build-arg IMAGE_ARG=$(IMG):$(TAG) -t $(IMG):$(TAG) .
```

Comment:
why --pull? we're using date-sha tags?

Reply:
bootstrap is based on `debian:jessie`, so we should always pull in updates before building it.

Reply:
er right...
/lgtm
/hold